src/theme/theme.tsx

```diff
@@ -1,83 +1,191 @@
-import { extendTheme } from "@chakra-ui/react";
+import { extendTheme } from '@chakra-ui/react';
 
 const theme = extendTheme({
   shadows: {
-    blueBorder: "inset 0px 0px 0px 3px rgba(145, 216, 246, 1)",
-    grayBorder: "inset 0px 0px 0px 3px rgba(221, 221, 221, 1)",
-    pressed: "inset 2px 4px 4px rgba(0, 0, 0, 0.35)",
+    blueBorder: 'inset 0px 0px 0px 3px rgba(145, 216, 246, 1)',
+    grayBorder: 'inset 0px 0px 0px 3px rgba(221, 221, 221, 1)',
+    pressed: 'inset 2px 4px 4px rgba(0, 0, 0, 0.35)',
+  },
+
+  colors: {
+    lightblue: '#91D8F6',
+    blue: '#22347F',
+    darkblue: '#001A33',
+    green: '#839E61',
+    darkgreen: '#01282B',
+    black: '#121212',
```

Comment:
can we test these custom colors bc i tried using them for the chakra progress bar on 3-curriculum and it didn't show up